packages/extension/public/css/daily-companion-app.css

```diff
@@ -8,4 +8,7 @@ daily-companion-app {
   position: fixed;
   z-index: 2147483647;
   font-feature-settings: normal!important;
```

Comment:
I'm going to work as a prettier here - let's insert a whitespace before !important  😀

Reply:
I think both variants are valid, but added a space 👍